modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/java/JavaErrorParsingTestCase.java

```diff
@@ -170,8 +165,9 @@ public void operationsWithErrorTypesButExtensionWithNone() throws Exception {
 
 
   @Extension(name = "Heisenberg")
-  @Operations({HeisenbergOperations.class, MoneyLaunderingOperation.class,
-      KillingOperations.class, HeisenbergScopes.class, HeisenbergRouters.class})
+  // @Operations({HeisenbergOperations.class, MoneyLaunderingOperation.class,
+  // KillingOperations.class, HeisenbergScopes.class, HeisenbergRouters.class})
+  @Operations(OperationWithThrows.class)
```

Comment:
Commented code

Reply:
removing the commented code, but removing the operations from this test extension is still the right thing to do